mail/protocols/smtp/src/main/java/com/fsck/k9/mail/transport/smtp/SmtpTransport.java

```diff
@@ -372,37 +373,22 @@ private void parseOptionalSizeValue(Map<String, String> extensions) {
 
     @Override
     public void sendMessage(Message message) throws MessagingException {
-        List<Address> addresses = new ArrayList<>();
-        {
-            addresses.addAll(Arrays.asList(message.getRecipients(RecipientType.TO)));
-            addresses.addAll(Arrays.asList(message.getRecipients(RecipientType.CC)));
-            addresses.addAll(Arrays.asList(message.getRecipients(RecipientType.BCC)));
+        Set<String> addresses = new LinkedHashSet<>();
```

Comment:
Using a `Set` will fix the send error due to duplicate recipient addresses described in #4781.